### PR TITLE
Split security gloves from black gloves, gave sprites to those and tackling gloves, moves tackling to right-click

### PR DESCRIPTION
**Pull request by carlarctg**
## About The Pull Request

Added security gloves, which have the fast cuff trait. Normal black gloves no longer have them.

Adds sprites for security and tackling gloves, moves tackling to right-click!

Also added blue security gloves for the blue sec PR. Whoever merges it first will, assumedly, add it then.

Sprites here:
image

Moved tackling to right click.

## Why It's Good For The Game

> Added security gloves, which have the fast cuff trait. Normal black gloves no longer have them.

Didn't make any sense that normie gloves had these. It was an undocumented feature! Why can jannies fast cuff!

> Adds sprites for security and tackling gloves, moves tackling to right-click!

Really smells that these don't have sprites so I quickly slapped some up, edited from tackler and CE insuls.

> Also added blue security gloves for the blue sec PR. Whoever merges it first will, assumedly, add it then.

We can't have blue sec with red gloves!!!

> Moved tackling to right click.

It's so fuckin annoying to accidentally tackle when you're trying to throw something. It's the biggest thing that prevents me from putting on gloves. Now it'll rarely, if ever, be a problem: Pressing right click while thowing mode is on is unlikely to ever happen accidentally.

## Changelog

:cl:
add: Added security gloves, which have the fast cuff trait. Normal black gloves no longer have them.
imageadd: Adds sprites for security and tackling gloves, moves tackling to right-click!
imageadd: Also added blue security gloves for the blue sec PR. Whoever merges it first will, assumedly, add it then.
qol: Moved tackling to right click.
/:cl:

### CODE REVIEW

**Comment by alien3301** [comment]
Do you mean tackling is right click with empty hand or throw mode empty hand with right click? If it's just rightclick it's gonna be even worse than before imo

**Comment by Jacquerel** [comment]
> Do you mean tackling is right click with empty hand or throw mode empty hand with right click? If it's just rightclick it's gonna be even worse than before imo

It says in the PR body "Right click while throwing mode is enabled"

**Comment by Wallemations** [comment]
> blue sec or

What where

**Comment by Sadboysuss** [comment]
Yes, this is so good.

**Comment by SomeoneYouProbablyKnow** [comment]
One unintentional consequences is that just getting black gloves no longer makes you secure from 90% of detectives. Now you can actually say "find a person with black gloves" and it means something. Which is neat

**Comment by necromanceranne** [comment]
Could you also throw on a combat mode check to the right click? That way, you can still shove reliably without having to worry about accidentally sending yourself hurtling into the thing you're trying to shove someone into.

**Comment by alien3301** [comment]
> Could you also throw on a combat mode check to the right click? That way, you can still shove reliably without having to worry about accidentally sending yourself hurtling into the thing you're trying to shove someone into.

You need throw mode AND right click. If you're trying to shove someone while having throw mode on that's a massive skill issue

**Comment by necromanceranne** [comment]
> You need throw mode AND right click. If you're trying to shove someone while having throw mode on that's a massive skill issue

Oh I'm silly.

**Comment by norsvenska** [comment]
just an fyi 'imageadd' should be replaced with just 'image' in the changelog

**Comment by kawaiinick** [comment]
detective will look dumb in these cause he isn't red

**Comment by SomeoneYouProbablyKnow** [comment]
> detective will look dumb in these cause he isn't red

Absolutely skill issue in the style department, use the epic Kim jacket, use the blue gloves, use darker clothes or just get regular gloves/latex gloves from the det vendor in your office, not like you should care about the 1 extra second on cuffs as det.

**Comment by Loafin34** [comment]
can you add blue gloves into the secdrobe vendor in the premium section? because blue is more royal than red?

**Comment by EuclidianGeometer** [comment]
Hm. I feel like black gloves making it harder to track you down because they're the same ones sec uses is intended, I'm not sure I like this change. Sprites are cute though.

**Comment by Sadboysuss** [comment]
> Hm. I feel like black gloves making it harder to track you down because they're the same ones sec uses is intended, I'm not sure I like this change. Especially re: the janitor, their ability to clean up their crimes/leave very little evidence seems pretty intended, and having the same gloves security does is part of that.  Sprites are cute though.

This still doesn't explain why all black gloves increase cuffing speed

**Comment by SomeoneYouProbablyKnow** [comment]
Most combat related gloves do, including some modsuit gloves i think. The original PR was more about reducing cuffing speed for everyone else. Part of the reasoning was to put it on an item partly and easily accessible to the crew and potential traitors. So actually saying this out loud, better give stationsiders some ghetto option for faster cuffing, else this would just be a sec buff mascarading as a fix.

**Comment by Sadboysuss** [comment]
> Most combat related gloves do, including some modsuit gloves i think. The original PR was more about reducing cuffing speed for everyone else. Part of the reasoning was to put it on an item partly and easily accessible to the crew and potential traitors. So actually saying this out loud, better give stationsiders some ghetto option for faster cuffing, else this would just be a sec buff mascarading as a fix.

It doesn't make sense for the color black to just give you faster cuffing speed. A ghetto alternative is probably fine

**Comment by SkeletalElite** [comment]
> > Most combat related gloves do, including some modsuit gloves i think. The original PR was more about reducing cuffing speed for everyone else. Part of the reasoning was to put it on an item partly and easily accessible to the crew and potential traitors. So actually saying this out loud, better give stationsiders some ghetto option for faster cuffing, else this would just be a sec buff mascarading as a fix.
> 
> It doesn't make sense for the color black to just give you faster cuffing speed. A ghetto alternative is probably fine

The ghetto tackle gloves are an option. They have fast cuffing. Presumably the reason the black gloves are faster and other colors aren't is the same reason they're fireproof and the others aren't.

**Comment by SomeoneYouProbablyKnow** [comment]
I think ghetto tackle gloves are also fingerless so you'll leave your kidnapping prints on everything as you go. Can't really think of an existing alternative.

**Comment by Sadboysuss** [comment]
> The ghetto tackle gloves are an option. They have fast cuffing. Presumably the reason the black gloves are faster and other colors aren't is the same reason they're fireproof and the others aren't.

This brings me to the next question, why is gloves being black - make them fireproof?

**Comment by carlarctg** [comment]
> detective will look dumb in these cause he isn't red

your job isnt to arrest people

**Comment by carlarctg** [comment]
> Most combat related gloves do, including some modsuit gloves i think. The original PR was more about reducing cuffing speed for everyone else. Part of the reasoning was to put it on an item partly and easily accessible to the crew and potential traitors. So actually saying this out loud, better give stationsiders some ghetto option for faster cuffing, else this would just be a sec buff mascarading as a fix.

no lol

**Comment by carlarctg** [comment]
> > > Most combat related gloves do, including some modsuit gloves i think. The original PR was more about reducing cuffing speed for everyone else. Part of the reasoning was to put it on an item partly and easily accessible to the crew and potential traitors. So actually saying this out loud, better give stationsiders some ghetto option for faster cuffing, else this would just be a sec buff mascarading as a fix.
> > 
> > 
> > It doesn't make sense for the color black to just give you faster cuffing speed. A ghetto alternative is probably fine
> 
> The ghetto tackle gloves are an option. They have fast cuffing. Presumably the reason the black gloves are faster and other colors aren't is the same reason they're fireproof and the others aren't.

ill remoev that

**Review by Wallemations — APPROVED** [review]
The sprites look fine but I continue to not be an advocate for blue sec

File: code/modules/clothing/gloves/color.dm [hunk 1]
```diff
@@ -20,6 +19,15 @@
...
 	)
 
+/obj/item/clothing/gloves/color/black/security
+	name = "security gloves"
+	desc = "These security gloves come with microchips that help the user quickly restrain suspects."
+	icon_state = "sec"
+	clothing_traits = list(TRAIT_FAST_CUFFING)
+
+/obj/item/clothing/gloves/color/black/security/blu
+	icon_state = "sec_blu"
```
Comment (san7890) [inline code/modules/clothing/gloves/color.dm:29]: why not `blue`

**Review by san7890 — CHANGES_REQUESTED** [review]

File: code/modules/clothing/gloves/color.dm (hunk 1 above)
Comment (carlarctg) [reply code/modules/clothing/gloves/color.dm:29]: tf2

Comment (necromanceranne) [reply code/modules/clothing/gloves/color.dm:29]: Blutarch Mann's legacy lives on in these shitty themed gloves.

Comment (san7890) [reply code/modules/clothing/gloves/color.dm:29]: silly but fine

**Comment by DrDiasyl** [comment]
> Moved tackling to right click.
For some reason, you can still tackle with a left click and it fucked me up today.

**Comment by DrDiasyl** [comment]
Found out, #86031 accidentally deleted it as there is nothing in changelog or PR body stating about deleting the right-click check.

**Pull request merged**